test/chemistry/test_qeom_ee.py

```diff
@@ -24,6 +25,8 @@
 from qiskit.chemistry.core import Hamiltonian, TransformationType, QubitMappingType
 from qiskit.chemistry.algorithms import QEomEE
 
+# TODO Update this test after excited states PR is done
```

Comment:
I am not sure why this test would need updating since QEomEE will be deprecated right - same for QEomVQE test below. As such this test will remain while the components remain and be deleted later. For sure new unit tests will be needed for the new excited states but I see this test (and the EOMVQE equivalent) remaining.

Reply:
removed the TODOs and I agree that the test should stay. Resolved